packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -202,60 +209,115 @@ interface NonEmptyRowProps {
   childPath: string;
   schema: JsonSchema;
   rowIndex: number;
+  moveUp: () => any;
+  moveDown: () => any;
+  isLast: boolean;
+  sortButtons: boolean;
 }
 
-const NonEmptyRow = React.memo(({
-  childPath,
-  schema,
-  rowIndex,
-  openDeleteDialog
-}: NonEmptyRowProps & WithDeleteDialogSupport) => (
-    <TableRow key={childPath} hover>
-      {generateCells(NonEmptyCell, schema, childPath)}
-      <NoBorderTableCell style={styles.fixedCell}>
-        <div style={{ display: 'flex', justifyContent: 'center' }}>
-          <IconButton
-            aria-label={`Delete`}
-            onClick={() => openDeleteDialog(childPath, rowIndex)}
-          >
-            <DeleteIcon />
-          </IconButton>
-        </div>
-      </NoBorderTableCell>
-    </TableRow>
-  ));
+const NonEmptyRow = React.memo(
+  ({
+    childPath,
+    schema,
+    rowIndex,
+    openDeleteDialog,
+    moveUp,
+    moveDown,
+    isLast,
+    sortButtons
+  }: NonEmptyRowProps & WithDeleteDialogSupport) => {
+    return (
+      <TableRow key={childPath} hover>
+        {generateCells(NonEmptyCell, schema, childPath)}
+        <NoBorderTableCell
+          style={sortButtons ? styles.fixedCell : styles.fixedCellSmall}
+        >
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
```

Comment:
I think the div should be removed as the Grid below is handling the correct item placement now